functions.lua

```diff
@@ -298,6 +298,24 @@ function PacifistMod.remove_misc()
         assert(entry[1] and entry[2])
         data_raw.remove(entry[1], entry[2])
     end
+
+    -- hide artillery flare
+    data_raw.hide("artillery-flare", "artillery-flare")
+end
+
+function PacifistMod.hide_all_remnants()
+    data_raw.hide_all("corpse", data_raw.get_all_names_for({"corpse"}))
```

Comment:
I was thinking that only combat would destroy buildings. It makes sense that vehicle collisions and mods like reactor explosions could as well.

The main thing I was trying to fix was that remnants of military buildings showed up in the rich text selection, probably because they were orphaned by removing the military entities.

Reply:
oh, reactor explosions are in the base game. Let your reactor heat up to 1000°C, steer a tank into it. BOOM.